v0/destinations/woopra/data/WoopraPageConfig.json

```diff
@@ -0,0 +1,91 @@
+[
+    {
```

Comment:
This is the same mapping from identify. Rather here, map the fields mentioned [here](https://rudderlabs.slack.com/archives/C03THKDK7GF/p1662619819297989?thread_ts=1662574807.867089&cid=C03THKDK7GF). Some of them are available in generic fields as well.

If we have provision to merge user data with these, then lets use the constructPayload function twice for the two mapping JSONS and merge the payload generated in a single final payload.

Reply:
Means to map some already defined (at Woopra ) events to ours and take a custom one as well just in case ?

Reply:
- The fields that are stated in woopra [doc](https://docs.woopra.com/reference/track-ce#pageviews) for page call, are already expected fields from [rudderstack page call properties](https://www.rudderstack.com/docs/event-spec/standard-events/page/#properties). 
- So, for page call, let us create a dedicated mapping JSON, where these values will be mapped from properties  and context.page object as I showed here. If any field beyond these comes in properties, lets send it out with proper prefix.

<img width="777" alt="Screenshot 2022-09-08 at 8 27 04 PM" src="https://user-images.githubusercontent.com/60211312/189155717-b7557775-845a-4bc1-98bd-4ed14c2b6793.png">

Reply:
for Page call we can use either of the below 
const eventName = type === "page" ? "Loaded a Page" : "Loaded a Screen";
This one is for Mixpanel.

OR

{
    "destKey": "cv_event",
    "sourceKeys": ["event", "properties.event", "name", "properties.name"],
    "required": true,
    "metadata": {
      "defaultValue": "Web Page Viewed",
      "template": "Web Page Viewed: {{value}}"
    }
  }

we can use this as well, found under Clevertap.